CefSharp.Wpf/IME/IMEWpfKeyboardHandler.cs

```diff
@@ -0,0 +1,305 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Windows.Input;
+using System.Windows.Interop;
+using CefSharp.Structs;
+using CefSharp.Wpf.Internals;
+using CefSharp;
+
+namespace CefSharp.Wpf.IME
+{
+    public class IMEWpfKeyboardHandler : WpfKeyboardHandler
+    {
+        int _languageCodeId;
+        bool _systemCaret;
+        bool _isDisposed = false;
+        List<Rect> _compositionBounds = new List<Rect>();
+        HwndSource _source;
+        Func<IntPtr, int> _loWord;
+
+        internal bool IsActive { get; set; }
+
+        /// <summary>
+        /// The source hook
+        /// </summary>
+        private HwndSourceHook sourceHook;
+
+        public IMEWpfKeyboardHandler(ChromiumWebBrowser owner) : base(owner)
+        {
+            if (IntPtr.Size == sizeof(Int32))
+                _loWord = x => x.ToInt32();
+            else
+                _loWord = LOWORD64;
```

Comment:
Instead of a function reference, calling your function should be fine. It could test `IntPtr.Size`, and the JIT compiler would optimize the test.